Detections/AuditLogs/UseraddedtoPrivilgedGroups.yaml

```diff
@@ -0,0 +1,41 @@
+id: 4d94d4a9-dc96-410a-8dea-4d4d4584188b
+name: User added to Azure Active Directory Privileged Groups
+description: |
+  'This will alert when a user is added to any of the Privileged Groups.
+  For further information on AuditLogs please see https://docs.microsoft.com/azure/active-directory/reports-monitoring/reference-audit-activities.
+  For Administrator role permissions in Azure Active Directory please see https://docs.microsoft.com/azure/active-directory/users-groups-roles/directory-assign-admin-roles'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1098
+  - T1078
+query: |
+  let timeframe = 1h;
```

Comment:
Are we sure we want to run this hourly?

Reply:
Yes, i think user added to admin is crucial and should be run hourly to alert potential persistence at early stage rather than day frequency.